neo/io/blackrockio.py

```diff
@@ -622,7 +623,7 @@ def __read_nev_header(self, ext_header_variants):
             ('nb_ext_headers', 'uint32')]
 
         nev_basic_header = np.fromfile(filename, count=1, dtype=dt0)[0]
-        print(nev_basic_header) ###
+        #print(nev_basic_header) ###
```

Comment:
In our version of fixes, we removed this print statement entirely. I think both is fine, however, if logging, maybe one should identify first the best places to put logger.debug() statements.